{"name": "Get user", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'idfdtg' in {'avatar': 'https://reqres.in/img/faces/1-image.jpg', 'email': 'george.bluth@reqres.in', 'first_name': 'George', 'id': 1, ...}", "trace": "self = <main.TestApi object at 0x0000019F4A585070>\n\n    @allure.title(\"Get user\")\n    @allure.description(\"Getting user by id\")\n    @allure.severity(allure.severity_level.NORMAL)\n    @allure.feature(\"GET\")\n    def test_single_user(self):\n        with allure.step(\"Step 1. Get base url\"):\n            url = self.base_url + '1'\n        with allure.step(\"Step 2. Request\"):\n            res = requests.get(url)\n        with allure.step(\"Step 3. Assert that status code should equal 200\"):\n            assert res.status_code == 200\n        with allure.step(\"Step 4. Assert that content type should be application/json; charset=utf-8\"):\n            assert res.headers['Content-Type'] == 'application/json; charset=utf-8'\n        with allure.step(\"Step 5. Convert response to json. Get data\"):\n            user = res.json()['data']\n        with allure.step(\"Step 6. Assert that id exist\"):\n>           assert 'idfdtg' in user\nE           AssertionError: assert 'idfdtg' in {'avatar': 'https://reqres.in/img/faces/1-image.jpg', 'email': 'george.bluth@reqres.in', 'first_name': 'George', 'id': 1, ...}\n\nmain.py:24: AssertionError"}, "description": "Getting user by id", "steps": [{"name": "Step 1. Get base url", "status": "passed", "start": 1711035200213, "stop": 1711035200213}, {"name": "Step 2. Request", "status": "passed", "start": 1711035200213, "stop": 1711035200324}, {"name": "Step 3. Assert that status code should equal 200", "status": "passed", "start": 1711035200325, "stop": 1711035200325}, {"name": "Step 4. Assert that content type should be application/json; charset=utf-8", "status": "passed", "start": 1711035200325, "stop": 1711035200325}, {"name": "Step 5. Convert response to json. Get data", "status": "passed", "start": 1711035200325, "stop": 1711035200325}, {"name": "Step 6. Assert that id exist", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'idfdtg' in {'avatar': 'https://reqres.in/img/faces/1-image.jpg', 'email': 'george.bluth@reqres.in', 'first_name': 'George', 'id': 1, ...}\n", "trace": "  File \"D:\\Univ\\8 sem\\TestAutomation\\code\\pr10\\main.py\", line 24, in test_single_user\n    assert 'idfdtg' in user\n"}, "start": 1711035200325, "stop": 1711035200325}], "start": 1711035200213, "stop": 1711035200325, "uuid": "0164d532-d09b-4687-97cc-6b861c85c3aa", "historyId": "bdfa703b11356bd6ca7a8074e2cf5fb6", "testCaseId": "bdfa703b11356bd6ca7a8074e2cf5fb6", "fullName": "main.TestApi#test_single_user", "labels": [{"name": "epic", "value": "Api testing"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "GET"}, {"name": "suite", "value": "main"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-FUOKO33"}, {"name": "thread", "value": "2480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "main"}]}